src/VPolytope.jl

```diff
@@ -501,15 +501,26 @@ Return an `VRep` polyhedron from `Polyhedra.jl` given a polytope in V-representa
 - `backend` -- (optional, default: `default_polyhedra_backend(P, N)`) the polyhedral
                computations backend, see [Polyhedra's documentation](https://juliapolyhedra.github.io/Polyhedra.jl/latest/installation.html#Getting-Libraries-1)
                for further information
+- `relative_dimension` -- (default, optional: `nothing`) an integer representing
```

Comment:
What is the *relative* dimension?

Reply:
```julia
julia> p = polyhedron(Polyhedra.vrep([[0.0, 0.0], [1.0, 1.0]])) # a line segment
Polyhedron DefaultPolyhedron{Float64,Polyhedra.Intersection{Float64,Array{Float64,1},Int64},Polyhedra.Hull{Float64,Array{Float64,1},Int64}}:
2-element iterator of Array{Float64,1}:
 [0.0, 0.0]
 [1.0, 1.0]

julia> Polyhedra.dim(p)
1

julia> p = polyhedron(Polyhedra.vrep([[0.0, 0.0], [1.0, 1.0], [0.0, 2.0]])) # a triangle
Polyhedron DefaultPolyhedron{Float64,Polyhedra.Intersection{Float64,Array{Float64,1},Int64},Polyhedra.Hull{Float64,Array{Float64,1},Int64}}:
3-element iterator of Array{Float64,1}:
 [0.0, 0.0]
 [1.0, 1.0]
 [0.0, 2.0]

julia> Polyhedra.dim(p)
2
```
Here by "relative" i mean relative to the given set.

Reply:
I am afraid that this term is not universally understood. Especially since it is used in the error message. Maybe you can elaborate in the docs using this example and explicitly pointing out that a line segment has relative dimension one even if the ambient dimension is two.